templates/dynamic/landing.html

```diff
@@ -65,14 +65,21 @@ <h2 class="sr-only">TaxBrain</h2>
                     </td>
                     <td>
                       {% flatblock "dynamic_behavioral_blurb" %}
+                      Blue bottle chia keffiyeh, master cleanse pork belly fanny pack kitsch tousled. Everyday carry offal wolf chartreuse asymmetrical, yuccie twee chambray shabby chic roof party selfies fingerstache listicle chia.
                     </td>
                 </tr>
                 {% if is_authenticated %}
                 <tr padding="15px">
                     <td>
-                    <a href="/dynamic/ogusa/{{pk}}/?start_year={{start_year}}" class="text-white btn btn-secondary">Overlapping Generations Simulation</a>
+                    <a href="/dynamic/ogusa/{{pk}}/?start_year={{start_year}}" class="text-white btn btn-secondary disabled">Overlapping Generations Simulation</a>
                     </td>
                     <td>
+                      <div class="construction">
+                        <div class="progress progress-striped" style="min-width: 400px; width: 100%">
+                          <div class="progress-bar active progress-bar-custom" role="progressbar" aria-valuenow="45" aria-valuemin="0" aria-valuemax="100" style="width: 100%"></div>
+                        </div>
+                        <small>Under Construction - Coming Soon!</small>
```

Comment:
I can stick this, "Under Construction - Coming Soon!" in the "dynamic_olg_blurb" flatblock.